web/test/test_utils.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+
+from bfex.common.utils import *
+
+
+class TestFacultyNames(object):
+    def test_validate_name(self):
+        valid_name = "JNelson.Amaral"
+        invalid_name = "Nelson Amaral"
+        invalid_name_2 = "Nelson, Amaral"
+
+        assert FacultyNames.validate_name(valid_name)
+        assert not FacultyNames.validate_name(invalid_name)
+        assert not FacultyNames.validate_name(invalid_name_2)
+
+    def test_build_directory_url(self):
+        valid_name = "Nelson.Amaral"
+        valid_name_2 = "JNelson.Amaral"
+
+        url_name = FacultyNames.build_url_name(valid_name)
+        assert url_name.islower()
+        assert "-" in url_name
+        assert "nelson-amaral"
```

Comment:
assert "nelson-amaral" doesn't really check anything, does it?
something to check instead could be assert "nelson-amaral" == url_name ?